requirements.txt

```diff
@@ -65,6 +63,10 @@ Django==1.11.4
 djangorestframework==3.6.3
 django-cors-headers==1.3.1
 djangorestframework-bulk==0.2.1
+# django-celery-beat==1.0.1  # BSD 3 Clause
+# Contains a fix for handling disabled tasks that still has not been release
+git+git://github.com/celery/django-celery-beat@f014edcb954c707cb7628f4416257b6a58689523# BSD 3 Clause
```

Comment:
Looks like 1.1.0 was released a couple weeks ago, which contains this commit. Can we update?

Btw, this isn't a blocker; if you can confirm that we can just upgrade, then that can be done in a follow-up PR.

Reply:
I don't know of any reason why we would not be able to.